packages/client/src/api/oAuth/index.ts

```diff
@@ -0,0 +1,89 @@
+import { BaseApi } from '@api/baseApi'
+import { ClientID, OauthSignInRequest } from './types'
+import { Routes } from '@routes/constants'
+import {
+    generateRandomStateStr,
+    removeState,
+    saveState,
+} from '@utils/authentication/oauth'
+import { closePopup } from '@utils/ui/popup'
+
+const BaseURL = 'oauth'
```

Comment:
Здесь и ниже все константы стоит назвать либо в SCREAMING_SNAKE_CASE либо в camelCase. Первая большая буква у типов, интерфейсов и енамов обычно, поэтому не очень очевидно может быть

```suggestion
const BASE_URL = 'oauth'
```